Sources/TuistGenerator/Descriptors/ProjectDescriptor.swift

```diff
@@ -0,0 +1,27 @@
+import Basic
+import Foundation
+import XcodeProj
+
+public class ProjectDescriptor {
+    /// Path to the project
+    public var path: AbsolutePath
```

Comment:
Nitpick: `manifesPath`?

Reply:
in this case it's the project path, not so much the path of the manifest file (as with the `WorkspaceDescriptor`

Reply:
In that case let's leave it as `path`.